README.md

```diff
@@ -156,6 +156,10 @@ const contentful = require("contentful");
 const contentful = require("contentful/dist/contentful.browser.min.js");
 ```
 
+#### Angular universal:
```

Comment:
Can you move this section under advanced?

Reply:
Alright, I took the react stuff too and moved them both over